snooty/tinydocutils/states.py

```diff
@@ -1103,6 +1109,14 @@ def check_subsection(self, source: str, style: StyleKind, lineno: int) -> bool:
             if style.overline is not None:
                 memo.section_bubble_up_kludge = True
 
+            if level <= self.memo.minimum_level_stack[-1]:
+                self.parent.append(
+                    self.reporter.error(
+                        "Block content cannot contain sections that are sibling to the parent element's section level",
```

Comment:
So I understand what this message means, but I'm not sure our users will. Can we provide an example of when this happens and how to remediate?

Reply:
It's tricky -- I would like to have a more detailed error message with an example, but historically we have short single-line error messages, with an exception for the rare suggested correction, and any explanation will be vertically very long.

What do you think about a link to a page to explain the issue in more detail, perhaps?

Reply:
That sounds *great*!

Reply:
https://github.com/mongodb/snooty-parser/wiki/Section-Hierarchy-Must-Be-Linear

How does this look?